airflow/models/__init__.py

```diff
@@ -188,6 +188,55 @@ def get_fernet():
     return _fernet
 
 
+class XComSerializer(types.TypeDecorator):
```

Comment:
I'd suggest to directly move this new class out into it's own file?
See discussion https://lists.apache.org/thread.html/3c15a871b15f99b4dd4bef25085b4de1c167c728f08262df73ef7217@%3Cdev.airflow.apache.org%3E

Reply:
Hey @seelmann, thanks for the suggestion. That's a good idea. Wanted to check if it also makes sense to move the XCom model / class to this file as well (i.e. airflow/models/xcom.py would contain the XCom class and XComSerializer class).